etcdmgmt/utils.go

```diff
@@ -0,0 +1,66 @@
+package etcdmgmt
+
+import (
+	"io/ioutil"
+	"path"
+
+	"github.com/coreos/etcd/embed"
+	"github.com/ghodss/yaml"
+	config "github.com/spf13/viper"
+)
+
+// EtcdConfigFile is path to etcd config file
+var EtcdConfigFile string
+
+// EtcdMinimalConfig represents essential etcd parameters.
+type EtcdMinimalConfig struct {
```

Comment:
They are already set in `GetNewEtcdConfig` and also inside `embed.NewConfig()`. Also, the things left out remain same for standalone mode and also when glusterd2 is part of the cluster. Only initial cluster, cluster state and name is required to reconfigure etcd to make it join a cluster.

`EtcdMinimalConfig` is a subset of `EtcdConfigReq`